templates/profile_edit.html

```diff
@@ -30,3 +32,47 @@
     <button type="submit" value="Submit" class="submit"> Save Changes</button>
 </form>
 {% endblock content %}
+
+
+{% block javascript %}
+  <script>
+    $("#edit_profile").submit(function(event) {
+        event.preventDefault();
+        user_id = {{ user_info_dict.user_id }};
```

Comment:
define `data` var here and add parameters in it:
```javascript
data: {
  'pk': '{{ user_info_dict.user_id }}',
}
```

Reply:
that is already defined within AJAX function.

Reply:
so define it here and just use it as assignment inside the ajax function.